README.md

```diff
@@ -57,6 +57,7 @@ Just like all linuxdeploy plugins, the Qt plugin's behavior can be configured so
 **Qt specific:**
 - `$QMAKE=/path/to/my/qmake`: use another `qmake` binary to detect paths of plugins and other resources (usually doesn't need to be set manually, most Qt environments ship scripts changing `$PATH`)
 - `$EXTRA_QT_PLUGINS=pluginA;pluginB`: Plugins to deploy even if not found automatically by linuxdeploy-plugin-qt
+  - example: `$EXTRA_QT_PLUGINS=svg;` if you are using SVG icons
```

Comment:
You put it too simple. It needs to basically say "use `svg` if you want to use the module [`QtSvg`](https://doc.qt.io/qt-5/qtsvg-index.html)".

Reply:
I'd would want to keep the `EXTRA_QT_PLUGINS=svg;` part to make it perfect clear what to put in your build script for the impatient 😉

Reply:
We could combine both, right? You can combine a short demo with an elaborate explanation on how this actually works.

Reply:
Like my last commit?